### PR TITLE
Improvements suggestions

### PR DESCRIPTION
**Pull request by riacataquian**
Here are various improvements regarding styleguides and wordings. Would love feedbacks!

### CODE REVIEW

File: lib/koans/09_functions.ex [hunk 1]
```diff
@@ -47,20 +47,20 @@ defmodule Functions do
     assert bigger(4, 27) == ___
   end
 
-  def the_length(0), do: "It was zero"
-  def the_length(number), do: "The length was #{number}"
+  def get_number(0), do: "The number was zero"
+  def get_number(number), do: "The number was #{number}"
```
Comment (felipesere) [inline lib/koans/09_functions.ex]: I don't know what I was thinking when I named these functions. 👍

**Comment by felipesere** [comment]
Awesome! Thanks for your suggestions.
Do you mind looking through the other koans to check for inconsistent indentation and comma-in-list-missplacements?

File: lib/koans/11_processes.ex [hunk 2]
```diff
@@ -81,8 +81,8 @@ defmodule Processes do
   koan "Processes can be used to hold state" do
     initial_state = "foo"
     pid = spawn(fn ->
-      state(initial_state)
-    end)
+            state(initial_state)
```
Comment (ukutaht) [inline lib/koans/11_processes.ex]: I have to say that to my eyes it's more readable without the change :shrug:

File: lib/koans/06_maps.ex [hunk 3]
```diff
@@ -2,13 +2,13 @@ defmodule Maps do
   use Koans
 
   @person %{
-    name: "Jon",
+    first_name: "Jon",
```
Comment (ukutaht) [inline lib/koans/06_maps.ex]: :+1:

**Comment by riacataquian** [comment]
@felipesere Sure, no problem.

File: lib/koans/11_processes.ex (hunk 2 above)
Comment (riacataquian) [inline lib/koans/11_processes.ex]: Is that so? I'm willing to undo it. Do you want me to? :)

Comment (ukutaht) [inline lib/koans/11_processes.ex]: Yeah, I'd say let's undo it if you're OK with that :)

**Comment by riacataquian** [comment]
Also, I have a question @felipesere and @ukutaht. Should we be using Elixir's convention when referring to functions?

Please see: https://github.com/elixirkoans/elixir-koans/blob/master/lib/koans/13_agents.ex#L14
I'd suggest it to be: `To update the state of an agent, use the Agent.update/2 function`. I'm having some doubts. Thanks!

**Comment by felipesere** [comment]
I am not overly picky on `Agent.update/2` vs. `update`. 😸 
Once we have more infrastructure in place, I'd prefer to push the descriptions (of the modules) and the individual koans more towards story-telling than API-describing. 📚

**Comment by riacataquian** [comment]
Sure @felipesere thanks for the input! 👍

**Pull request merged**